Dockerfile

```
@@ -1,13 +1,40 @@
-FROM golang:1.18 AS build
+ARG REG=docker.io
+FROM ${REG}/library/golang:1.17 AS build
 WORKDIR /src
-ENV GO111MODULE=on
-COPY . /src
-RUN CGO_ENABLED=0 go build -o main && go test ./... -v
+RUN go install github.com/swaggo/swag/cmd/swag@v1.7.1
```

Project is using `v1.8.0` right now
```suggestion
RUN go install github.com/swaggo/swag/cmd/swag@v1.8.0
```

---

Gosh good call on all these version mismatches. Nice catch!